### PR TITLE
Docs ssh instructions output

### PR DESCRIPTION
**Pull request by kstohr**
## Status

Ready for review 

## Description of Changes
Changes proposed in this pull request:
https://github.com/freedomofpress/securedrop/issues/3348

## Testing
make docs-lint - passed

### CODE REVIEW

**Comment by dachary** [comment]
Would you mind only including commit **added line clarifying terminal output vs terminal command** in this PR?

**Comment by dachary** [comment]
@rebased against develop to resolve the conflict.

File: docs/servers.rst [hunk 1]
```diff
@@ -352,3 +352,5 @@ the below commands. You should not be prompted for a passphrase
     app
     $ ssh <username>@<Monitor IP address> hostname
     mon
+
+If you have successfully connected to the app via SSH, the terminal output will be name of the app to which you have connected ('app' or 'mon') as shown above.
```
Comment (redshiftzero) [inline docs/servers.rst]: Let's replace `app` with `server` in its first two uses, as the word app is also the hostname of one of the two servers. So we'd have: `If you have successfully connected to the server via SSH, the terminal output will be name of the server to which you have connected ('app' or 'mon') as shown above.`

**Comment by redshiftzero** [comment]
Good news, your changes were merged in #3591 - thanks for your contribution!

**Pull request closed without merging**